awxkit/test/cli/test_options.py

```diff
@@ -128,17 +128,17 @@ def test_integer_argument(self):
         page = OptionsPage.from_json({
             'actions': {
                 'POST': {
-                    'limit': {'type': 'integer'}
+                    'max_hosts': {'type': 'integer'}
                 },
             }
         })
-        options = ResourceOptionsParser(None, page, 'job_templates', self.parser)
+        options = ResourceOptionsParser(None, page, 'organizations', self.parser)
```

Comment:
I decided to change this, because using `job_templates` was causing the unit tests to traverse through this new `OPTIONS /api/v2/job_templates/N/launch` code, and I didn't feel like mocking it all out.